matter_server/server/device_controller.py

```diff
@@ -1067,8 +1067,42 @@ async def _setup_node(self, node_id: int) -> None:
             return
         self._nodes_in_setup.add(node_id)
         node_logger = LOGGER.getChild(f"node_{node_id}")
-        try:
-            async with self._node_setup_throttle:
+        node_data = self._nodes[node_id]
+        log_timers: dict[int, asyncio.TimerHandle] = {}
+
+        def log_node_long_setup(time_start: float) -> None:
+            """Temporary measure to track a locked-up SDK issue in some (special) circumstances."""
+            time_mins = int((time.time() - time_start) / 60)
+            node_model = node_data.attributes.get(
+                "0/40/14", node_data.attributes.get("0/40/3", "")
```

Comment:
Can we add a comment what these numbers mean exactly?

Reply:
Its the modelname from the raw attributes ;-)

Reply:
added comment